src/js/node/http2.ts

```diff
@@ -3509,6 +3663,39 @@ class Http2Server extends net.Server {
   }
 }
 
+Http2Server.prototype[EventEmitter.captureRejectionSymbol] = function (err, event, ...args) {
+  switch (event) {
+    case "stream": {
+      const { 0: stream } = args;
+      if (stream.sentHeaders) {
+        stream.destroy(err);
+      } else {
+        stream.respond({ [HTTP2_HEADER_STATUS]: 500 });
+        stream.end();
+      }
+      break;
+    }
+    case "request": {
+      const { 1: res } = args;
+      if (!res.headersSent && !res.finished) {
+        // Don't leak headers.
+        for (const name of res.getHeaderNames()) {
+          res.removeHeader(name);
+        }
+        res.statusCode = 500;
+        res.end(STATUS_CODES[500]);
+      } else {
+        res.destroy();
+      }
```

Comment:
```suggestion
      args.unshift(err, event);
      ReflectApply(net.Server.prototype[EventEmitter.captureRejectionSymbol], this, args);
```

should these be uncommented?

Reply:
Not for this PR, need to wait the net refactor